backend/src/services/auth/auth.service.ts

```
@@ -1152,9 +1155,43 @@ export class AuthService {
         break;
       }
 
+      case 'apple': {
+        let appleUserInfo;
+        try {
+          appleUserInfo = await this.appleOAuthProvider.verifyIdToken(
+            idToken,
+            audience
+          );
```

ESLint

[error] Replace `⏎············idToken,⏎············audience⏎··········` with `idToken,·audience`

(prettier/prettier)